rfcs/text/0001-rfc-process.md

```diff
@@ -103,6 +161,22 @@ Where necessary, existing RFCs can be revised using the same process (a proposed
   
      ***@channel**: Call for community review of the RFC process - https://github.com/HumanCellAtlas/dcp-community/pull/27 - Last call is April 1.*
 
+**ADDITION BEGINS**
+
```

Comment:
Editorial: I'd suggest re-formatting this section to merge the recommendations. For example:

For software RFCs, it is **strong recommended** that:


- An FYI be sent to the HumanCellAtlas **#tech-architecture** slack channel. Include a link to the RFC pull request and the last call deadline.
- The RFC is added as an agenda item to the next *DCP Architecture* meeting for spirited discussion

Reply:
Re-formatted!